server/src/main/java/org/apache/kafka/server/metrics/ClientMetricsConfigs.java

```diff
@@ -143,7 +148,7 @@ private static void validateProperties(Properties properties) {
 
         // Make sure that client match patterns are valid by parsing them.
         if (properties.containsKey(CLIENT_MATCH_PATTERN)) {
```

Comment:
Though doesn't matter much but can we have consistency please as above we have used `parsed.containsKey` and here `properties.containsKey`.

Reply:
Sorry for this mistake. Fixed.